src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -41,60 +37,68 @@ public enum armPosition {
   // setting up CAN IDs for the motors
   public Arm(int rightMotorCANID, int leftMotorCANID) {
 
+    //rightmotor.resetfactorydefualts
+    //leftmotor.resetfactorydefualts
+
     // motor type for right motor
     this.rightMotor = new CANSparkMax(rightMotorCANID, MotorType.kBrushless);
+    this.leftMotor = new CANSparkMax(leftMotorCANID, MotorType.kBrushless);
 
-    // comment below is for incase the motors need to be reversed
-    // this.rightMotor.setInverted(invertRightMotor);
+    this.armEncoder = new Encoder(EncoderPort1, EncoderPort2);
```

Comment:
We're missing the import for Encoder (`edu.wpi.first.wpilibj.Encoder` I believe) and there's nothing named `EncoderPort1` and `EncoderPort2`. Adding temporary values for these to constants would fit what is done in the rest of the project.